plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -166,6 +171,11 @@ public MachineProviderImpl(DockerConnector docker,
         this.doForcePullOnBuild = doForcePullOnBuild;
         this.privilegedMode = privilegedMode;
         this.snapshotUseRegistry = snapshotUseRegistry;
+        if (Strings.isNullOrEmpty(cheUserId)) {
+            this.cheUserId = System.getenv("CHE_USER_ID") == null ? "" : System.getenv("CHE_USER_ID");
```

Comment:
Could you, please, explain how to use the property when starting the che server using:

```
docker run ... eclipse/che-server
```